lib/endpoint/find-parameters.js

```diff
@@ -33,7 +33,7 @@ function findParameters (state) {
   findInBlocks(state)
   findInRoutePath(state)
   addPaginationParameters(state)
-
+  makeRelativeLinksAbsolute(state.results[0].params)
```

Comment:
sorry, nearly there :) You have to pass in `state` like the others do it, then iterate trough `state.results` & `state.results[].params`, to fix all the `state.results[].params[].descripiton` properties. 

Also please unto the white space changes. You removed empty lines 54, 76, 83, 91, 104, try to keep your changes as minimal as possible, that’s a general best practise :)